.env.development

```
@@ -1,4 +1,8 @@
+# .env.dev (dev project)
 VITE_SUPABASE_ANON_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImRtbXlyaXFibHRqcnR2dnBsbG16Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDY1NzY2OTAsImV4cCI6MjA2MjE1MjY5MH0.xw4zI0aDw9tYU7cJwSa9RcaE2nhl-juZpXTcnbsgfrU"
 VITE_SUPABASE_PROJECT_ID="dmmyriqbltjrtvvpllmz"
 VITE_SUPABASE_PUBLISHABLE_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImRtbXlyaXFibHRqcnR2dnBsbG16Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDY1NzY2OTAsImV4cCI6MjA2MjE1MjY5MH0.xw4zI0aDw9tYU7cJwSa9RcaE2nhl-juZpXTcnbsgfrU"
 VITE_SUPABASE_URL="https://dmmyriqbltjrtvvpllmz.supabase.co"
+VITE_ENVIRONMENT=development
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_ENVIRONMENT key should go before the VITE_SUPABASE_ANON_KEY key

(UnorderedKey)